src/main/java/com/compatibility/tinkers/ToolBrokenCheck.java

```diff
@@ -0,0 +1,27 @@
+package com.compatibility.tinkers;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import org.jetbrains.annotations.Nullable;
+
+public class ToolBrokenCheck
+{
+    private static final String STATS = "Stats";
+    private static final String BROKEN = "Broken";
+    public static boolean checkTinkersBroken(@Nullable ItemStack stack, @Nullable String tool)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "tool". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)